### PR TITLE
make redhat conversion tool downloadable with Pip

### PR DESCRIPTION
**Pull request by jasinner**
In order to install the Red Hat converter tool via Pip we need a setup.py. Adding that here.

This change avoids specifying the runtime dependencies in both Pipenv and setup.py by requesting users install the module locally with:

`pipenv install -e .`

As specified in the updated README.md

### CODE REVIEW

File: tools/redhat/setup.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+REQUIRES = [
+    "jsonschema",
+    "requests"
+]
+
+setup(
+    name="redhat_osv",
+    version="1.0.0",
+    description="Newtopia CLI v1",
```
Comment (andrewpollock) [inline tools/redhat/setup.py:15]: Copy pasta?

```suggestion
    description="Convert a Red Hat CSAF document to OSV format",
```

File: tools/redhat/setup.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    author_email="jshepher@redhat.com",
+    url="",
+    keywords=["Swagger", "Newtopia CLI v1"],
```
Comment (andrewpollock) [inline tools/redhat/setup.py:18]: Maybe this instead?

```suggestion
    keywords=["OSV", "JSON", "Conversion"],
```

File: tools/redhat/setup.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    install_requires=REQUIRES,
+    classifiers=[
+        "Programming Language :: Python :: 3",
+    ],
+    packages=find_packages(exclude=["*_test.py"]),
+    entry_points={"console_scripts": ["convert_redhat=convert_redhat"]},
+    long_description="""\
+    The purpose of this CLI is to interact with the manifest and dependency information API from the Deptopia project, and Manifest box repo.  # noqa: E501
```
Comment (andrewpollock) [inline tools/redhat/setup.py:26]: Fix copy pasta

```suggestion
    The purpose of this tool is to convert from CSAF documents to OSV # noqa: E501
```

**Review by andrewpollock — CHANGES_REQUESTED** [review]
Looks like there's a little bit of copy pasta left over?

**Comment by jasinner** [comment]
Sorry about that. I've updated the value to be specific to this module as per your suggestions.

**Comment by jasinner** [comment]
Justing putting this in Draft while I attempt to import it using Pip in another project.

**Comment by jasinner** [comment]
> Just putting this in Draft while I attempt to import it using Pip in another project.

Ok, I had to move things around a little bit, but this is importing successfully with Pip now. @andrewpollock can you please have another look and merge if you're happy.

**Review by andrewpollock — APPROVED** [review]

**Pull request merged**